_data/store/releases/TeamsIntegrationBundle.yaml

```
@@ -1,2 +1,2 @@
 versions:
-    - [1.0.0, 2.44.0]
+    - [1.1.0, 2.43.0]
```

Lets keep version history available
```suggestion
    - [1.1.0, 2.43.0]
    - [1.0.0, 2.44.0]    
```